packages/react-components/lib/map/three-fiber/shape-three-rendering.tsx

```diff
@@ -0,0 +1,85 @@
+import { Circle, Html } from '@react-three/drei';
+import React from 'react';
+
+interface ShapeThreeRenderingProps {
+  position: [number, number, number];
+  color: string;
+  text?: string;
+  circleShape: boolean;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const debounce = <T extends (...args: any[]) => void>(
```

Comment:
it looks like we're not using this function anywhere else at the moment, is it ok if we use
```
export const debounce = (callback: () => void, delay: number): (() => void) => {...}
```
This way we can avoid the eslint disabling

Reply:
Done 87b60ccd91cea03f780ba75f79d229b47a1edc8e